src/server/cluster/outgoing_slot_migration.cc

```diff
@@ -20,7 +20,10 @@
 #include "server/server_family.h"
 #include "util/fibers/synchronization.h"
 
-ABSL_FLAG(int, slot_migration_connection_timeout_ms, 5000, "Timeout for network operations");
+ABSL_FLAG(int, slot_migration_connection_timeout_ms, 2000,
```

Comment:
Have you synced with @romange about the new flag? I am asking cause we usually discuss before we introduce a new flag

Reply:
we have discussed it with @adiholden